src/Impostor.Server/Net/Inner/Objects/InnerPlayerControl.cs

```diff
@@ -383,7 +383,7 @@ public override async ValueTask<bool> HandleRpcAsync(ClientPlayer sender, Client
                         return false;
                     }
 
-                    Rpc44SetRole.Deserialize(reader, out var role);
+                    Rpc44SetRole.Deserialize(reader, out var role, out var _);
```

Comment:
by then, should we add a anticheat here to reject double set alive role for a client?

And we may prevent setting a player to ghost at round 1 since that will blackout all clients (have no idea how to do this)

Reply:
That seems out of scope for this PR, I've made a note in my todo list of future PR's